### DISCUSSION TITLE
Request for Thumbnail Color Override

### DISCUSSION THREAD

**Discussion by Durahl** [OrcaSlicer/OrcaSlicer#3560] [Ideas]
Greetings!

I was wondering if we could have a `Thumbnail Color Override` akin to the one found in **_SuperSlicer_**?

I mainly print my parts in black resulting in the Thumbnails being kinda hard to read on the dark Voron Themed UI of my Printer:
![IMG](https://i.imgur.com/gLX27hM.png)

Thanks for your consideration! 😁

**Comment by PatrickGlatz** [comment]
You can do this already when clicking on the front of your material (where there is this number "1" is mentioned) and select your desired color: (here it would print blue)
<img width="426" alt="image" src="https://github.com/SoftFever/OrcaSlicer/assets/26554156/4cd20943-822f-440f-810f-25094971da67">
